jenkins-pipelines/artifacts-azure-image.jenkinsfile

```diff
@@ -10,7 +10,8 @@ artifactsPipeline(
     backend: 'azure',
     instance_provision: 'spot',
 
-    timeout: [time: 45, unit: 'MINUTES'],
+    timeout: [time: 145, unit: 'MINUTES'],
```

Comment:
why do you need to much time? other artifacts have a 30 minutes timeout

Reply:
There's some instability in test that is hard to find the reason. Job is aborted before python times-out and logs are not showing any clue.
I hope to get some python traceback after increasing this timeout.